scripts/run_celery_sms.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -e
+
+celery -A run_celery.notify_celery sms --loglevel=INFO --concurrency=1 -queue send-sms-tasks
```

Comment:
the 'sms' here should just be 'worker', but with 'worker' it's the exact same command as in run_celery.sh, which means it just creates an identical instance. I'm not sure how to differentiate it.